src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -261,6 +261,7 @@
 
   <ItemGroup Label="Build Tools" Condition="$([MSBuild]::IsOsPlatform('Windows'))">
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.0.0" PrivateAssets="All" />
+    <PackageReference Include="Microsoft.CodeAnalysis.RulesetToEditorconfigConverter" Version="3.0.0" PrivateAssets="All" />
```

Comment:
This is cool, but how does it work? Does it pick up ANY ruleset file associated with the projects and merge it with editorconfig? 

If so, where are you referincing `Sdl.Recommended.Error.ruleset` ?